sys/include/net/ieee802154.h

```diff
@@ -112,28 +112,43 @@ extern const uint8_t ieee802154_addr_bcast[IEEE802154_ADDR_BCAST_LEN];
 /** @} */
 
 /**
+ * @defgroup net_ieee802154_conf     Default values for IEEE802.15.4
```

Comment:
Unifying with the other configuration groups

```suggestion
 * @defgroup net_ieee802154_conf     IEEE802.15.4 compile configurations
```

?

Reply:
roger